### PR TITLE
Book section merge

### PR DESCRIPTION
**Pull request by Gina-Frankel**
**Create a grid with bootstrap on the _DoublePage_ component**
image

row 1: title taking up 6/12 columns 
row 2: **two BookSection components** each component takes up 6/12 columns
row 3: **two BookSection components** each component takes up 6/12 columns

- Responsive for medium and large screens

This can be refactored into other components, we need to think about how we want to do this.

**BookSection components**
- If a prop called **line** has been passed down renders a border on the right side of the div
- Renders a box with: 
   -  background colour (at the moment coloured with the same colour as the book)
  -  border 
  - padding

**Other changes** 
- Move background-color to index sass from doublePage sass

### CODE REVIEW

File: src/components/chapterPages/chapterPages.tsx [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-import "bootstrap/dist/css/bootstrap.css"
-// import "./chapterPages.scss"
-
-export const ChapterPages = () => {
-  return (
-    <div className="d-flex justify-content-center" id="double-page">
-      <div id="centerLine"> </div>
-    </div>
-  )
-}
```
Comment (cjm106) [inline src/components/chapterPages/chapterPages.tsx:11]: I'm not sure we ever got the name right but still think we need this (or a version of ) to help reduce the complexity of the doublePage.tsx component.

File: src/components/doublePage/doublePage.tsx [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+        </div>
+      </div>
+
+      {/* row 3 */}
+      <div className="row">
+        <div className=" col-md-6 col-lg-6 book-mid-line ">
+          <BookSection />
+        </div>
+        <div className=" col-md-6 col-lg-6">
+          <BookSection />
```
Comment (cjm106) [inline src/components/doublePage/doublePage.tsx:34]: Great to get the postioning right!

Comment (cjm106) [reply src/components/doublePage/doublePage.tsx:34]: The code is quite repetitive - should the row be a component in its own right?

**Review by cjm106 — COMMENTED** [review]
I think we should merge this.

File: src/components/bookSection/bookSection.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import "bootstrap/dist/css/bootstrap.css"
+import "./bookSection.scss"
+
+export const BookSection = props => {
+  return (
+    <div className="  px-3 py-2">
```
Comment (benfurber) [inline src/components/bookSection/bookSection.tsx:7]: What's this spacing in the className doing here!?

File: src/components/bookSection/bookSection.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import "bootstrap/dist/css/bootstrap.css"
+import "./bookSection.scss"
+
+export const BookSection = props => {
```
Comment (benfurber) [inline src/components/bookSection/bookSection.tsx:5]: Are there any props?

File: src/components/doublePage/doublePage.scss [hunk 5]
```diff
@@ -11,12 +10,6 @@ h3 {
...
-#centerLine {
-  background-color: #9c9998;
-  width: 0.125%;
-}
-
-body {
-  background-color: #bcc2c9;
-  font-family: "Staatliches", cursive;
+.book-mid-line {
+  border-right: 1px solid black;
```
Comment (benfurber) [inline src/components/doublePage/doublePage.scss:14]: Let's have a global definition of black please.

File: src/components/doublePage/doublePage.tsx [hunk 6]
```diff
@@ -1,15 +1,37 @@
 import React from "react"
 import "bootstrap/dist/css/bootstrap.css"
 
-import { ChapterPages } from "../index"
+import { ChapterPages, BookSection } from "../index"
 import "./doublePage.scss"
 
 export const DoublePage = () => {
+  let customStyleRule = { "border-right": "1px solid black" }
```
Comment (benfurber) [inline src/components/doublePage/doublePage.tsx:8]: Doesn't look like this is used anywhere?

File: src/styles/variables.scss [hunk 7]
```diff
@@ -2,6 +2,7 @@ $text-color__primary: #111;
 $text-color__secondary: #ffffff;
 $book-page-color: #ffffff;
 $book-cover-color: #ffffff;
+$box-border-colour: #111;
```
Comment (benfurber) [inline src/styles/variables.scss:5]: So it's not worth having a global style if the variable is so specific. Why is the box-border this hex?

File: src/components/layout/index.scss [hunk 8]
```diff
@@ -7,6 +7,7 @@ body {
   margin: 0;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #bcc2c9;
```
Comment (benfurber) [inline src/components/layout/index.scss:10]: This should be a variable.

**Review by benfurber — CHANGES_REQUESTED** [review]

File: src/components/doublePage/doublePage.tsx (hunk 2 above)
Comment (Gina-Frankel) [reply src/components/doublePage/doublePage.tsx:34]: Yes agreed - it is very repetitive ... 
We could break the components into a **row** or maybe...

```
 <div className="row">
          <BookSection />
          <BookSection />
</div>
 <div className="row">
          <BookSection />
          <BookSection />
</div>
   
```
`<BookSection />` - would then contain `<div className=" col-md-6 col-lg-6 book-mid-line ">` combined with what it already contains ....

File: src/styles/variables.scss (hunk 7 above)
Comment (Gina-Frankel) [reply src/styles/variables.scss:5]: I think that many of our variables names are too specific like 
```
$book-page-color: #ffffff;
$book-cover-color: #ffffff;
```
I am thinking that it maybe should be like: 
$primary-color:
$secondary-color: 

I will have a look at all the names and think about better ways to name them ...

Comment (benfurber) [reply src/styles/variables.scss:5]: Great! That suggestion is a big improvement.

**Pull request merged**